src/SfxWeb/src/app/services/tree.service.ts

```diff
@@ -124,13 +124,27 @@ export class TreeService {
                     nodeId: IdGenerator.nodeGroup(),
                     displayName: () => 'Nodes',
                     selectAction: () => this.routes.navigate(() => nodes.viewPath),
-                    childrenQuery: () => this.getNodes(),
+                    childrenQuery: () => getNodesTypePromise,
                     badge: () => nodes.healthState,
                     listSettings: this.settings.getNewOrExistingTreeNodeListSettings(nodes.viewPath),
                     alwaysVisible: true
                 };
             }));
 
+            const getNodesTypePromise = this.data.getNodes().pipe(map(nodes => {
+                return nodes.nodeTypes.map(node => {
+                    return {
+                        nodeId: IdGenerator.node(node),
+                        displayName: () => node,
+                        selectAction: () => this.routes.navigate(() => nodes.viewTypePath(node)),
+                        childrenQuery: () => this.getNodes(node),
+                        badge: () => nodes.healthState,
```

Comment:
this would likely show an incorrect state right?

if we have 2 node types and in a node is in error in node type A then i think this would also show node type B as having an error.